src/config/dynamic-theme-fixes.config

```diff
@@ -933,6 +933,15 @@ CSS
 
 ================================
 
+inoreader.com
+
+CSS
+div.article_expanded{
+ background-color: rgb(31, 35, 38); !important;
```

Comment:
There should be only one `;` at the end of line. Please use 4 spaces for indentation. Check, how your fix works in Light mode, maybe you should replace `rgb(31, 35, 38)` with `${white}` template, so that `white` color (or any other value) could be configurable with app settings.

Reply:
Formatting fixed.
As for color - it looks appropriate with rgb(31, 35, 38) in both Light and Dark modes. Changing to ${white} makes background white on a dark page (in Light mode) which is not looking good